packages/realm/scripts/submit-analytics.mjs

```diff
@@ -16,6 +16,356 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
-export function collectPlatformData() {
-  throw new Error("Not yet implemented!");
+// Submits install information to Realm.
+//
+// Why are we doing this? In short, because it helps us build a better product
+// for you. None of the data personally identifies you, your employer or your
+// app, but it *will* help us understand what language you use, what Node.js
+// versions you target, etc. Having this info will help prioritizing our time,
+// adding new features and deprecating old features. Collecting an anonymized
+// application path & anonymized machine identifier is the only way for us to
+// count actual usage of the other metrics accurately. If we don’t have a way to
+// deduplicate the info reported, it will be useless, as a single developer
+// `npm install`-ing the same app 10 times would report 10 times more than another
+// developer that only installs once, making the data all but useless.
+// No one likes sharing data unless it’s necessary, we get it, and we’ve
+// debated adding this for a long long time. If you truly, absolutely
+// feel compelled to not send this data back to Realm, then you can set an
+// environment variable named REALM_DISABLE_ANALYTICS.
+//
+// Currently the following information is reported:
+// - What version of Realm is being installed.
+// - The OS platform and version which is being used.
+// - Node.js version numbers.
+// - JavaScript framework (React Native and Electron) version numbers.
+// - An anonymous machine identifier and hashed application path to aggregate
+//   the other information on.
+
+import fs from "node:fs";
+import { fileURLToPath } from "node:url";
+import path from "node:path";
+import process from "node:process";
+import https from "node:https";
+import os from "node:os";
+import console from "node:console";
+import { createHmac } from "node:crypto";
+import { Buffer } from "node:buffer";
+
+import machineId from "node-machine-id";
+import commandLineArgs from "command-line-args";
+import fse from "fs-extra";
```

Comment:
`fs-extra` and `command-line-args` are both missing as dependencies.

Reply:
`command-line-args` is in the root `package.json` and I guess we should move it. And we don't see the issue with `fs-extra` in the integration tests as `integration-tests/environments/electron/package.json` has it as a dependency. I'll add both to the `realm` package's dependencies.